.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 defaults: &defaults
   working_directory: ~/hig
   docker:
-    - image: circleci/node:8.2.1
+    - image: circleci/node:8.10-stretch
```

Comment:
Updated docker image to the latest Node LTS. I mainly wanted this to get `yarn@^1.x.x`